CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 ### Fixed
 
+- `0.18.0`: fixed the previous release which was broken by console `errors`, failing on the JS `import` statements `(Unexpected token...)` ([@driesd](https://github.com/driesd) in [#441](https://github.com/teamleadercrm/ui/pull/441))
```

Comment:
Q: why does this start with `0.18.0`? Is this to show in which release the bug was introduced?